whisper/.envdefault

```diff
@@ -40,7 +40,15 @@ PROMPT=
 # CUDA_VISIBLE_DEVICES=0
 
 # Number of threads per worker when running on CPU
-OMP_NUM_THREADS=4
+NUM_THREADS=4
 
 # Number of workers
 CONCURRENCY=2
+
+# WEBSOCKET PARAMETERS
+STREAMING_PORT=80
+
+# HTTP PARAMETERS
+ENABLE_STREAMING=true
+
+VAD=auditok
```

Comment:
Please move this up, in the relevant section (it's not about serving parameters, it's more related to the details of the STT)